www/src/components/searchbar-body.js

```diff
@@ -0,0 +1,225 @@
+      color: selected ? `white` : `black`,
+      backgroundColor: selected ? `#744C9E` : `white`,
+      padding: rhythm(.5),
+    }}>
+      <div css={{
+        display: `flex`,
+        justifyContent: `space-between`,
+      }}>
+        <div
+          css={{
+            fontFamily: typography.options.headerFontFamily.join(`,`),
+            fontWeight: `bold`,
+          }}
+          >
+            {hit.name}
+          </div>
+
+          <div
+            css={{
+              display: `flex`,
+              alignItems: `center`,
+              fontSize: rhythm(.5),
+            }}
+            >
+              {hit.humanDownloadsLast30Days}
+              <img width="10"
+                height="10"
+                css={{
+                  marginLeft: rhythm(.25),
+                  marginBottom: 0,
+                }}
+                src={selected ? DownloadsWhite : DownloadsBlack}></img>
+            </div>
+      </div>
+
+
+      <div css={{
+        fontSize: rhythm(.6),
+        paddingTop: rhythm(.25),
+        lineHeight: rhythm(.75),
+      }}>
+        {hit.description}
+      </div>
+
+      {/* <div css={{
+        fontSize: rhythm(.5),
+        color: `#D3D3D3`,
+      }}>
+        {hit.keywords.join(", ")}
+      </div> */}
+
+      {/* <div
+        css={{
+          display: `flex`,
+          paddingTop: rhythm(.25),
+        }}
+        >
+        <img
+          width="20"
+          height="20"
+          src={hit.lastPublisher.avatar}
+         />
+          <span css={{paddingLeft: rhythm(.25), fontSize: rhythm(.5), textTransform: `uppercase`}}>{hit.lastPublisher.name}</span>
+          <span css={{paddingLeft: rhythm(.25), fontSize: rhythm(.5)}}>{lastUpdated}</span>
+      </div> */}
+    </Link>
+  )
+}
+
+
+class SearchBar extends Component {
+  constructor(props){
+    super(props)
+    this.state = {searchState: {query: this.urlToSearch(), page: 1} }
+    this.updateHistory = debounce(this.updateHistory, updateAfter)
+  }
+
+
+  urlToSearch = () => {
+    return (this.props.history.location.search).slice(2);
+  }
+
+  // Old way
+  // onSearchStateChange = searchState => {
+  //   clearTimeout(this.debouncedSetState);
+  //   this.debouncedSetState = setTimeout(() => {
+  //     this.props.history.replace(`/packages?=${searchState.query}`)
+  //   }, updateAfter);]
+  //   this.setState({ searchState })
+  // }
+
+  // New way
+  updateHistory(value){
+    this.props.history.replace(`/packages?=${value.query}`)
+  }
+
+  onSearchStateChange(searchState){
+    this.updateHistory(searchState)
+    this.setState({ searchState })
+  }
+
+  render(){
+    return(
+      <div>
+        <InstantSearch
+          apiKey="f54e21fa3a2a0160595bb058179bfb1e"
+          appId="OFCNCOG2CU"
+          indexName="npm-search"
+          searchState={this.state.searchState}
+          onSearchStateChange={this.onSearchStateChange.bind(this)}
+          >
+          <Search searchState={this.state.searchState.query} />
```

Comment:
Add `<Configure attributesToRetrieve={[/* only required attributes*/]} />` here to avoid pulling down the readme etc. for every package, it will make it significantly faster